.changes/unreleased/tinymce-TINY-10574-2024-01-15.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Changed
+body: The `highlight_on_focus` option now defaults to true, adding a focus outline to every editor
```

Comment:
I guess we should still have a `.` at the end here. We used to have a rule about adding that.

Reply:
added